Modules/Dining/MITDiningHouseMealListViewController.m

```diff
@@ -152,9 +157,17 @@ - (void)updateCurrentlyDisplayedItems
             }
         }
         self.currentlyDisplayedItems = filteredItems;
-        [self updateNotificationLabel];
     }
+    
+    if (self.currentlyDisplayedItems.count > 0) {
+        self.tableView.tableFooterView = nil;
+    } else {
+        self.tableView.tableFooterView = [UIView new];
```

Comment:
"Those cells" == which cells?

How is the display message being shown? A label somewhere?